spec/helpers/articles_helper_spec.rb

```diff
@@ -0,0 +1,21 @@
+describe ArticlesHelper, type: :helper do
+  describe 'article_element_dataについて' do
+    context 'sub_headが引数に渡された場合' do
+      it 'H3タグが含まれるHTMLを生成する' do
+        expect(helper.article_element_data('sub_head', 'text')).to eq '<h3>text</h3>'
+      end
+    end
+    context 'descriptionが引数に渡された場合' do
+      it 'Pタグが含まれるHTMLを生成する' do
+        expect(helper.article_element_data('description', 'text')).to eq '<p>text</p>'
+      end
+    end
+    context 'imgが引数に渡された場合' do
+      let(:picture){ create(:picture)}
+      it 'imgタグが含まれるHTMLを生成する' do
+        expected = "<img class=\"img-responsive\" src=\"/uploads/picture/image/#{picture.id}/sample.jpg\" alt=\"Sample\" />"
```

Comment:
__[rubocop]__
* Line is too long. [124/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)